packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -4422,6 +4442,243 @@ addEventListener('fetch', event => {});`
 				)
 			);
 		});
+
+		it("should upload an asset manifest of the files in the directory specified by --experimental-assets", async () => {
```

Comment:
it's not super clear how this test and the previous one differ. Feels like they could be combined into a singe test which validates that both:
- `wrangler deploy` works when there is no worker source
- the manifest is uploaded

To me, they don't mean much on their own.

Reply:
ah whoops the first no worker source tests is redundant, removed it